src/main/java/frc/robot/commands/auto/PathPlannerCommand.java

```diff
@@ -8,60 +8,69 @@
+  
+  Alliance alliance;
+  
+  public PathPlannerCommand(ArrayList<PathPoint> waypoints, Drivetrain drive) {
+    this(new ArrayList<PathPlannerTrajectory>(Arrays.asList(PathPlanner.generatePath(
+      new PathConstraints(AutoConstants.kMaxAutoSpeed, AutoConstants.kMaxAutoAccel),
+      waypoints.get(0),
+      waypoints.get(1),
+      (PathPoint[]) Arrays.copyOfRange(waypoints.toArray(), 2, waypoints.size())
+    ))), 0, drive, true);
+  }
+  
+  public PathPlannerCommand(String pathGroupName, int pathIndex, Drivetrain drive){
+    this(PathGroupLoader.getPathGroup(pathGroupName), pathIndex, drive, true); 
+  }
+  
+  public PathPlannerCommand(String pathGroupName, int pathIndex, Drivetrain drive, boolean resetPose){
+    this(PathGroupLoader.getPathGroup(pathGroupName), pathIndex, drive, resetPose); 
+  }
+  
+  public PathPlannerCommand(List<PathPlannerTrajectory> pathGroup, int pathIndex, Drivetrain drive, boolean resetPose){
+    addRequirements(drive);
+    if (pathIndex < 0 || pathIndex > pathGroup.size() - 1) {
+      throw new IndexOutOfBoundsException("Path index out of range"); 
     }
+    System.out.println("Number of paths: " + pathGroup.size());
     
-    public PathPlannerCommand(List<PathPlannerTrajectory> pathGroup, int pathIndex, Drivetrain drive, boolean resetPose){
-        addRequirements(drive);
-        if (pathIndex < 0 || pathIndex > pathGroup.size() - 1){
-            throw new IndexOutOfBoundsException("Path index out of range"); 
-        } 
-        
-        PathPlannerTrajectory path = pathGroup.get(pathIndex);
-
-        addCommands(
-            (pathIndex == 0 && resetPose ? new InstantCommand(() -> {drive.setPigeonYaw(path); drive.resetOdometry(path.getInitialHolonomicPose());}) : new DoNothing()),
-            new PrintCommand("Number of paths: " + pathGroup.size()),
-            new PPSwerveControllerCommand(
-                path, 
-                drive::getPose, // Pose supplier
-                drive.getPathplannerXController(), // X controller can't normal PID as pathplanner has Feed Forward 
-                drive.getPathplannerYController(), // Y controller can't normal PID as pathplanner has Feed Forward 
-                drive.getPathplannerRotationController(), // Rotation controller can't normal PID as pathplanner has Feed Forward 
-                (chassisSpeeds) -> { drive.setChassisSpeeds(chassisSpeeds, false); }, // chassis Speeds consumer
-                true,  // use Alliance color
-                drive // Requires this drive subsystem
-            )
-        );
-    }
+    addCommands(new InstantCommand( () -> {
+      PathPlannerTrajectory path = PathPlannerTrajectory.transformTrajectoryForAlliance(
+        pathGroup.get(pathIndex),DriverStation.getAlliance());
+      if (resetPose) {
+        drive.setPigeonYaw(path); 
+        drive.resetOdometry(Conversions.absolutePoseToPathPlannerPose(
+        path.getInitialHolonomicPose(), DriverStation.getAlliance()));
```

Comment:
Isn't odometry using an absolute pose? And the input parameter into the conversion is a pathplanner pose? So it should be Conversions.pathPlannerPoseToAbsolutePose(), right?